nautobot/docs/models/extras/jobbutton.md

```diff
@@ -0,0 +1,166 @@
+# Job Buttons
+
+Job Buttons are predefined buttons that allow users to run jobs directly from within Nautobot object views. It uses the object where the button was pressed as the only input to the job. These are helpful when you want to start a job that requires minimal or no input without having to use the standard job form. For example, you may have a job that only requires a user to select a device. Instead, they can now go to that device in the web UI and click the associated Job Button instead.
+
+Job Buttons can be created in web UI located in the navbar under Jobs > Job Buttons. Each button can be associated with multiple Nautobot object types (site, device, prefix, etc.) and will be displayed on all of the associated object detail views. The text displayed on the button supports Jinja2 templating which allows for using [context data](#context-data) to dynamically update or [even be hidden under certain conditions](#conditional-rendering).
+
+The buttons appear at the top right corner of an object's individual detail page for each object type they are associated to. They can be either individual buttons or grouped together in a dropdown for better organization. In either case, the order of the buttons are influenced by the `weight` number configured for each button.
+
+## Configuration
+
+* **Name** - A unique name for the Job Button.
+* **Object type(s)** - The type or types of Nautobot object that the button will be associated to.
+* **Text** - The text that will be displayed on the button.
+* **Job** - The [Job Button Receiver](#job-button-receivers) that this button will run.
+* **Weight** - The number used for determining the order the buttons will appear.
+* **Group** - The name of the dropdown group to add this button into (optional).
+* **Button Class** - The button CSS class, which dictates the color.
+* **Confirmation** - Should the button pop up a confirmation dialog before running.
+
+!!! warning
```

Comment:
I would think we should default safe for both commit and confirmation meaning

confirmation = True
commit = False

Reply:
after speaking with @joewesch I think the only actual request is to update the `help_text` of confirmation to be more clear on the implication of turning that off.